core/commonMain/src/ArgParser.kt

```diff
@@ -84,6 +81,9 @@ abstract class Subcommand(val name: String): ArgParser(name) {
  */
 class ArgParserResult(val commandName: String)
 
+@Deprecated("OPTION_PREFIX_STYLE is deprecated. Please, use OptionPrefixStyle.")
+typealias OPTION_PREFIX_STYLE = ArgParser.OptionPrefixStyle
```

Comment:
Is it on top-level because a typealias cannot be nested? Can we suppress the limitation and make it nested anyway?

Reply:
How can I suppress it? This is an error `Nested and local type aliases are not supported`.

Reply:
Try to suppress it the following way: `@Suppress("TOPLEVEL_TYPEALIASES_ONLY")`